docs/dev-guide.md

```diff
@@ -85,56 +85,34 @@ Commands `make dockers` and `make dockers_dev` are similar. The main difference
 ```
 In the command above each successive file overrides the previous parameters.
 
-A practical example in our case would be to add persistent volumes to the users-db, things-db, influxdb and grafana containers so that we don't loose data between updates of Mainflux. The volumes are mapped to the default location on the host machine, something like `/var/lib/docker/volumes/project-name_volume-name`:
+A practical example in our case would be to enable debugging and tracing in NATS so that we can see better how are the messages moving around.
 
+`docker-compose.nats-debugging.yml`
 ```yaml
-# docker/docker-compose.persistence.yml
 version: "3"
 
-volumes:
-  mainflux-users-db-volume:
-  mainflux-things-db-volume:
-
 services:
-  users-db:
-    volumes:
-      - mainflux-users-db-volume:/var/lib/postgresql/data
-
-  things-db:
-    volumes:
-      - mainflux-things-db-volume:/var/lib/postgresql/data
+  nats:
+    command: --debug -DV
 ```
 
-```yaml
-# docker/addons/influxdb-writer/docker-compose.persistence.yml
-version: "3"
-
-volumes:
-  influxdb-volume:
-  grafana-volume:
-
-services:
-  influxdb:
-    volumes:
-      - influxdb-volume:/var/lib/influxdb
-
-  grafana:
-    volumes:
-      - grafana-volume:/var/lib/grafana
-```
 When we have the override files in place, to compose the whole infrastructure including the persistent volumes we can execute:
 ```
-docker-compose -f docker/docker-compose.yml -f docker/docker-compose.persistence.yml up -d
-docker-compose -f docker/addons/influxdb-writer/docker-compose.yml -f docker/addons/influxdb-writer/docker-compose.persistence.yml up -d
+docker-compose -f docker/docker-compose.yml -f docker/docker-compose.nats-debugging.yml up -d
 ```
 
-__Note:__ Please store your customizations to some folder outside the Mainflux's source folder and maybe add them to some other git repository. You can always apply your customizations by pointing to the right file using `docker-compose -f ...`. Also be sure not to use the `make cleandocker` and `make cleanghost` as they might delete something which you don't want to delete (i.e. the newly created persistent volumes).
+__Note:__ Please store your customizations to some folder outside the Mainflux's source folder and maybe add them to some other git repository. You can always apply your customizations by pointing to the right file using `docker-compose -f ...`.
+
+
+### Cleaning up your dockerized Mainflux setup
+If you want to clean your whole dockerized Mainflux installation you can use the `make cleandocker` command. Please note that __by default the `make cleandocker` command will stop and delete all of the containers, images and persistent volumes__. If you want to keep gathered data in the system (the persistent volumes) please use the following command `make spv=true cleandocker` (spv = skip persistent volumes). This form of the command will stop and delete the containers and images, but won't delete the persistent volumes.
```

Comment:
I think it is better to skip persisten volume by default, but add explicitly extra parameter if you want to delete it also - like `pv=true`.

I think this can help avoinding errors and making fool-proof solution. I see more a situations where I cleaned dockers, but I fortgot vlumes - so I run the command again with correct param, no big deal. However, the other way around, if I cleaned everything because I forgot a param - it is too late :).

Reply:
@drasko I completely agree with you from data persistence view point. My idea was to to leave the current `make cleandocker` behavior as it is and offer an option to skip the deletetion of the persistent volumes.

On the second thought, taking in account all the complaints about data loss, I would by default skip the volume deletion.

I hope the possible running of the same command twice by the @mainflux/contributors  won't be a problem. 

I will prepare a new pr